frigate/ffmpeg_presets.py

```diff
@@ -51,7 +51,7 @@ def get_selected_gpu(self) -> str:
         return ""
 
 
-LIBAV_VERSION = int(os.getenv("LIBAVFORMAT_VERSION_MAJOR", "59") or "59")
+LIBAV_VERSION = int(os.getenv("LIBAVFORMAT_VERSION_MAJOR"))
```

Comment:
I think I'd still like there to be a default just in case

Reply:
Please let me try convincing you first:

1. We control all Frigate images, therefore `LIBAVFORMAT_VERSION_MAJOR` should be set in all situations
2. Setting a fallback makes it more difficult to spot issues. What if the person is providing an older `ffmpeg` but because of the `LIBAVFORMAT` was not detected, Frigate ended up using the wrong args?
3. Or even to help spot issues in the Frigate images themselves. If there was no such fallback, [this bug](https://github.com/blakeblackshear/frigate/pull/16712#discussion_r1964536619) would not have happened as it would have failed during config creation.

If that didn't change your mind, I'm happy to restore this change. Let me know.

Reply:
@NickM-27 sorry, tests failed, and I could not figure out an easy way to fix that. I added the fallback again.

Reply:
@NickM-27, I also refactored the way how the Python files read this variable, by consolidating it  into `const.py`. You may want to have a look at https://github.com/blakeblackshear/frigate/pull/16712/commits/ed1f10e6a426c8e8d33cb53ae057b8a81c65a0df.